### PR TITLE
feat: CRUD operations

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
- add note
- edit note
- update note
- delete note

Screenshot (81)

### CODE REVIEW

File: src/pages/NotePage/Notes.jsx [hunk 1]
```diff
@@ -1,13 +1,86 @@
...
 const expandCard=()=>{
     setNoteCard(true);
 }
+const date = new Date();
+const initialVal= {
+    _id:"",
+    title:"",
+    body:"",
+    backGround:"#e7dcdc",
+    CreatedAt:`${date.getDate()}/0${
```
Comment (abhisheksunil2201) [inline src/pages/NotePage/Notes.jsx]: Adding a 0 before every month can cause the double digit months to displayed as 12/010/2022. You could change it to some other approach. refer this https://bobbyhadz.com/blog/javascript-get-month-and-date-in-2-digit-format

File: src/pages/NotePage/Notes.jsx [hunk 2]
```diff
@@ -1,13 +1,86 @@
...
+
+const {token}= useAuth();
+const addNoteHandler=async()=>{
+   
+   try{
+       if(!note._id)
+       {
+        const response= await postNote({token,note});
+        
+        if(response.status===200 || response.status===201)
```
Comment (abhisheksunil2201) [inline src/pages/NotePage/Notes.jsx:39]: The if conditions and the else condition are the same except the postNote and editNote parts. so you could extract out the common functionality into a separate function.

**Review by abhisheksunil2201 — APPROVED** [review]
Have recommended a few changes. The code quality is good and it can be merged after the changes

**Review by ramalingaa — APPROVED** [review]
Code looks clean , try to use index imports 
This can be merged

**Pull request merged**